packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.tsx

```diff
@@ -0,0 +1,130 @@
+import { Component, Host, State, h } from '@stencil/core';
+import arrowRightSvg from '../../assets/arrow-right-white.svg';
+import { CONTACTS } from '../../../../react-components/src/components/Telephone/contacts';
+
+/**
+ * @componentName Crisis Line Modal
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+
+@Component({
+  tag: 'va-crisis-line-modal',
+  styleUrl: 'va-crisis-line-modal.scss',
+  shadow: true,
+})
+export class VACrisisLineModal {
```

Comment:
Can we have some tests written for this component?

Reply:
Good idea, on it!